scripts/setup-velox-torcharrow.sh

```diff
@@ -79,7 +79,7 @@ function wget_and_untar {
 wget_and_untar https://github.com/gflags/gflags/archive/refs/tags/v2.2.2.tar.gz gflags
 wget_and_untar https://ftp.openssl.org/source/openssl-1.1.1k.tar.gz openssl &
 wget_and_untar https://boostorg.jfrog.io/artifactory/main/release/1.69.0/source/boost_1_69_0.tar.gz boost &
-wget_and_untar https://github.com/facebook/folly/archive/v2023.12.04.00.tar.gz folly &
```

Comment:
Lets remove this file, as I dont think its being used any more.

Reply:
I can do it, but let me do it as a follow up PR to make them more atomic.

Reply:
https://github.com/facebookincubator/velox/pull/8961